sherpa/csrc/endpoint.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright      2022  (authors: Pingfeng Luo)
+ *
+ * See LICENSE for clarification regarding multiple authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef ENDPOINT_H_
```

Comment:
Please use
```bash
cd /path/to/sherpa
./scripts/check_style_cpplint.sh
./scripts/check_style_cpplint.sh 1
./scripts/check_style_cpplint.sh 2
```
to fix the code style issues.

For the style of header guard, we are using
```
SHERPA_CSRC_FILENAME_H_
```

----

In your case, it should be
```
SHERPA_CSRC_ENDPOINT_H_
```